auditing/auditing-interceptor.go

```diff
@@ -334,9 +337,10 @@ func NewConnectInterceptor(a Auditing, logger *zap.SugaredLogger, shouldAudit fu
 	}
 }
 
-func HttpFilter(a Auditing, logger *zap.SugaredLogger) restful.FilterFunction {
+func HttpFilter(a Auditing, logger *slog.Logger) restful.FilterFunction {
```

Comment:
Would be great to change the signature so the function returns an error so the caller has to handle the case if the `a` parameter is nil.

Reply:
done